openedx2zim/entrypoint.py

```diff
@@ -202,6 +202,20 @@ def main():
         default=1,
     )
 
+    parser.add_argument(
+        "--watcher-min-dl-count",
+        help="Minimum number of resources to have downloaded before considering to stop on errors",
```

Comment:
```suggestion
        help="[dev] Minimum number of resources to have downloaded before considering to stop on errors",
```

Just to make sure users understand it's not common usage